README.md

```diff
@@ -14,6 +14,18 @@ The servers are written in Typescript.
   A simple server that accepts websocket connections and sends pre-set transcriptions to the client.
 * [Bot API](ac-bot-api) \[[API documentation](https://techdocs.audiocodes.com/voice-ai-connect/#VAIG_API/API_1.htm)\]:
   A server that echos incoming messages. It also supports pre-defined replies like "disconnect", which hangs up.
+  To use the bot, you need to set the environment variable ACCESS_TOKEN for running the bot, and configure it accordingly in VoiceAI Connect.
+  For example, if the token is MyS3cr3tT0k3n, the configuration should look like this:
+  ```json
+  {
```

Comment:
Put it inside `providers` field with an array.

Reply:
Done